src/Infrastructure/Loader/TaskYaml/TaskYamlFileLoader.php

```diff
@@ -0,0 +1,189 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Sdk\Infrastructure\Loader\TaskYaml;
+
+use SprykerSdk\Sdk\Infrastructure\Builder\TaskSet\TaskFromYamlTaskSetBuilderInterface;
+use SprykerSdk\Sdk\Infrastructure\Builder\TaskYaml\TaskBuilderInterface;
+use SprykerSdk\Sdk\Infrastructure\Collector\TaskYamlCollector;
+use SprykerSdk\Sdk\Infrastructure\Dto\TaskYamlCriteriaDto;
+use SprykerSdk\Sdk\Infrastructure\Storage\InMemoryTaskStorage;
+use SprykerSdk\SdkContracts\Entity\PlaceholderInterface;
+use SprykerSdk\SdkContracts\Entity\TaskInterface;
+
+class TaskYamlFileLoader implements TaskYamlFileLoaderInterface
```

Comment:
Would be fine to move  methods: `getTaskPlaceholders`, `isTaskIdExist` to the `TaskStorageInterface`.
We shouldn't keep taskStorage as a service. We need to be sure that the taskStorage is always being populated. Need to create Registry that returns this storage and keeps it always loaded. We need one point when we can get any task and this point should be always valid.